haystack/document_store/sql.py

```diff
@@ -326,6 +326,12 @@ def write_labels(self, labels, index=None):
 
         labels = [Label.from_dict(l) if isinstance(l, dict) else l for l in labels]
         index = index or self.label_index
+        ids_exist_in_db: list = []
+        labels_found = self.get_labels_by_id(ids=[label.id for label in labels], index=index)
+        if len(labels_found) > 0:
+            ids_exist_in_db = [label.id for label in labels_found]
+            logger.warning(f"Duplicate labels: Label with ids {','.join(ids_exist_in_db)}' already exists in index "
```

Comment:
I think we need to be a bit clearer with this message.

1) It's not necessarily always the case that the Labels are exact duplicates, it might just be that they have the same ID because the user populated `label.id` with a question id instead of an annotation id
2) This message should say what is happening when we try to insert these Labels
3) I think IDs should come at the end of the string since we might have many many labels in a big dataset

I would propose something like:
```
f"Duplicate Label IDs: Inserting a Label whose id already exists in this document store. This will overwrite the old Label. Please make sure Label.id is a unique identifier of the answer annotation and not the question. Problematic ids: {','.join(ids_exist_in_db)}"
```

Reply:
Seems good to me. I will replace the message.